### ISSUE TITLE
Система просмотра оригиналов документов

### ISSUE THREAD

**Issue by Limych** [Limych/GeniBase#7]
Онлайн вьюер с возможностью ссылаться на конкретную часть любой страницы.

**Comment by a-KaDeMi-k** [comment]
да, очень пригодится для варианта ссылок на скан РАЗВОРОТА, когда на нём сразу 2 страницы

**Comment by Limych** [comment]
Эта система ещё не реализована — не закрывай этот issue!
